bin/docker_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
What's this file for? Naive search I don't see it used anywhere.

Reply:
For now I'm using it run tests in docker locally. I'll drop it from this PR, I'll figure it out in another PR what to do with it.